locals.tf

```diff
@@ -91,3 +91,14 @@ locals {
 
   docker_machine_adds_name_tag = signum(sum(local.docker_machine_version_test)) <= 0
 }
+
+resource "local_file" "config_toml" {
+  content  = local.template_runner_config
+  filename = "${path.module}/debug/runner_config.toml"
+}
+
+resource "local_file" "user_data" {
+  count    = var.enable_user_data_diffing ? 1 : 0
+  content  = nonsensitive(local.template_user_data)
+  filename = "${path.module}/debug/user_data.sh"
```

Comment:
suggestion: Include the `/debug` folder in `.gitignore` so we don't check in the files by accident.